update Readme with Mixins (#70)
Lightning-AI/models: README.md

```

</details>

## Model Registry Mixins

Lightning Models provides mixin classes that simplify pushing models to and pulling models from the registry.

> These mixins can be integrated directly into your model classes.

Reviewer: Usually we avoid using "your" in these cases. example it can be someone else's model and user might think they can't use the mixin from third party model. 

Also would be great adding a brief explainer of what is "Mixin".

### Available Mixins

1. **PickleRegistryMixin**: For serializing any Python class with pickle
2. **PyTorchRegistryMixin**: For PyTorch models, preserving both weights and constructor arguments

Using these mixins provides several advantages:

> - Direct integration into your model classes

Author: ```suggestion
- Direct integration into the model classes
```

- Simplified save/load workflow
- Automatic handling of model metadata and constructor arguments
- Version management support

> ### Using PickleRegistryMixin

Author: ```suggestion
### Using `PickleRegistryMixin`
```

> Add the mixin to your class for seamless registry integration:

Author: ```suggestion
Add the mixin to a Python class for seamless registry integration:
```

```python
from litmodels.integrations.mixins import PickleRegistryMixin

class MyModel(PickleRegistryMixin):
    def __init__(self, param1, param2):
        self.param1 = param1
        self.param2 = param2
        # Your model initialization code

# Create and push a model instance
model = MyModel(param1=42, param2="hello")
model.push_to_registry(name="my-org/my-team/my-model")

# Later, pull the model
loaded_model = MyModel.pull_from_registry(name="my-org/my-team/my-model")
```

> ### Using PyTorchRegistryMixin

Author: ```suggestion
### Using `PyTorchRegistryMixin`
```
